pyop2/base.py

```diff
@@ -917,7 +984,7 @@ def _process_args(cls, sets, **kwargs):
         try:
             sets = as_tuple(sets, ExtrudedSet)
         except TypeError:
-            sets = as_tuple(sets, Set)
+            sets = as_tuple(sets, (Set, type(None)))
```

Comment:
`type(None)` is usually called `NoneType`, but you need `from types import NoneType` to write that.